docs-v2/content/contributing/setting-up-a-dev-environment.md

```diff
@@ -1,64 +1,78 @@
-# Setting up a dev environment (React)
-
-This guide is here to help you get a local version of Storybook running so that you can hack on it
-and contribute back to the community.
+# Setting up a dev environment
 
 ## Prerequisites
 
 Please have the **_latest_** stable versions of the following on your machine
 
 -   node
--   npm
 -   yarn
--   create-react-app
-
-## Setting it up
+-   @storybook/cli
 
-### 1.  Setup storybook locally
+## Initial Setup
 
-If you run into trouble here, make sure your node, npm, and **_yarn_** are on the latest versions.
+If you run into trouble here, make sure your node, npm, and **_yarn_** are on the latest versions (yarn at least v1.0.0).
 
 1.  `cd ~` (optional)
 2.  `git clone https://github.com/storybooks/storybook.git` _bonus_: use your own fork for this step
 3.  `cd storybook`
 4.  `yarn`
 5.  `yarn bootstrap --core`
 6.  `yarn test --core`
+7. `yarn dev` *You must have this running for your changes to show up*
 
-#### Bootstrapping everything
+### Bootstrapping everything
 
 This method is slow, and if you are reading this tutorial you probably don't want to do it
 
 1.  `yarn bootstrap`
-2.  At the prompt: `a`, `enter`, `y`, `enter`
+2.  At the prompt: `a`, `enter`, `y`, `enter` ( `a` selects all (fully bootstrap) and `y` is to confirm)
 3.  Have a beer 🍺
-4.  `yarn test`
+4.  `yarn test` (to verify everything worked)
 
-### 2. Create a project (skip this if you already have a react project)
 
-1.  `cd ~` (optional)
-2.  `create-react-app storybook-sandbox-app`
-3.  `cd storybook-sandbox-app`
-4.  `yarn start`
-5.  Verify that you have a working react app.
+## Linking Storybook
```

Comment:
I think this section (and the one below it) should be moved to "Working with your own app"

Reply:
Agreed